tests/installation/bootloader_svirt.pm

```diff
@@ -212,7 +213,9 @@ sub run {
         my $cmdline = '';
         $cmdline .= 'textmode=1 ' if check_var('VIDEOMODE', 'text');
         $cmdline .= 'rescue=1 ' if is_installcheck || is_rescuesystem;    # rescue mode
-        $cmdline .= get_var('EXTRABOOTPARAMS') . ' ';
+        $cmdline .= get_var('EXTRABOOTPARAMS') . ' ' if get_var('EXTRABOOTPARAMS');
+        $cmdline .= registration_bootloader_cmdline if check_var('SCC_REGISTER', 'installation');
```

Comment:
I guess a better way would be to save a common string variable for all bootloader implementations and call `type_string` or the corresponding call like here. Can you try that?

Reply:
I don't have all backends to justify that it works. We have specific_bootmenu_params in bootloader_setup, which is called by most bootloaders but is still different here and there. So we may think about addressing it in future and unifying bootloaders more to avoid problems like one here, but it's not 1 hour task.